posts/models.py

```diff
@@ -180,7 +180,7 @@ def get_ext_feed(self, requestor):
         if self.trailing_slash:
             url = self.api + '/author/posts/'
         else:
-            url = self.api + '/author//posts'
```

Comment:
I am upset that I am pretty sure I wrote this line

Reply:
lmao it happens